README.md

```diff
@@ -159,9 +164,10 @@ LARAVEL_LOG_KEEPER_REMOTE_PATH           = "myproject1-prod-01"
 
 ## Security
 
-If you discover any security related issues, please email mathiasgrimm@gmail.com instead of using the issue tracker.
```

Comment:
this is actually dangerous. Security problems should not be reported publicly in the first moment